web/src/template/InterviewRecordingPage/InterviewRecordingPage.tsx

```diff
@@ -24,6 +24,7 @@ interface InterviewRecordingPageTemplateProps {
   progressbarCount: number;
   totalQuizCount: number;
   quiz: Quiz;
+  interview: Interview | undefined;
```

Comment:
Would
```
interview?: Interview 
```
look cleaner?

Reply:
Agreed, I forgot using undefined counts as it being optional as well 👍